graphsdk/src/androidTest/java/com/microsoft/graph/serializer/CalendarSerializerTests.java

```diff
@@ -0,0 +1,23 @@
+package com.microsoft.graph.serializer;
+
+import android.test.AndroidTestCase;
+
+import java.util.Calendar;
+
+
+/**
+ * Test cases for {@see CalendarSerializer}
+ */
+public class CalendarSerializerTests extends AndroidTestCase {
+
+    public void testCalendarSerializer() throws Exception {
```

Comment:
If we do not need any more coverage, please just delete this file and leave ISO8601Test.java alone

---
In reply to: [65755264](https://github.com/microsoftgraph/msgraph-sdk-android/pull/17#discussion_r65755264) [](ancestors = 65755264)